packages/react/package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "@humeai/voice-react",
-  "version": "0.1.7",
+  "version": "0.1.8",
```

Comment:
since we release versions in lock-step, need to update voice, embed, and embed-react as well

Reply:
done 👍